core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/PoolImpl.java

```diff
@@ -656,4 +656,10 @@ public boolean isSaved() {
     return getId() != UNSAVED_ID;
   }
 
+  @Override
+  public Set<String> getPrioritySubprojectAliases() {
+    return poolDilutions.stream()
+        .map(PoolDilution::getPoolableElementView).filter(view -> view.getSubprojectPriority() != null && view.getSubprojectPriority())
+        .map(view -> view.getSubprojectAlias()).collect(Collectors.toSet());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)